conn-fix/README.md

```diff
@@ -0,0 +1,359 @@
+
+# Determines if milliseconds should be added to timestamp fields. Only available for AML 3
+millisecondsInTimeStampFields: true
+
+# Determines if microseconds should be added to timestamp fields. Only available for AML 3. This value is more priority than 'Milliseconds In Time Stamp Fields'
+microsecondsInTimeStampFields: false
+
+# Receive limit in bytes to emulate Slow Consumer
+receiveLimit: 0
+
+# Client send reject if message is invalid
+rejectInvalidMessage: true          
+
+# If set to unchecked, user defined fields will not be rejected if they are not defined in the data dictionary, or are present in messages they do not belong to.
+validateUserDefinedFields: true
+
+# If set to unchecked, fields that are out of order (i.e. body fields in the header, or header fields in the body) will not be rejected. Useful for connecting to systems which do not properly order fields.
+validateFieldsOutOfOrder: true
+
+# If set to unchecked, messages which contain values not presented in the range will not be rejected.
+validateFieldsOutOfRange: true
+
+# If set to unchecked, fields without values (empty) will not be rejected. Useful for connecting to systems which improperly send empty tags.
+validateFieldsHaveValues: true
+
+# If set to checked, no reject sent on incoming message with duplicate tags Allow duplicated tags.
+duplicateTagsAllowed: false
+
+# If specified, the session start and end will be converted from default zone to UTC
+useLocalTime: false
+
+# Enables SSL usage for QFJ acceptor or initiator
+useSSL: false
+
+# KeyStore to use with SSL
+sslKeyStore: ""
+
+# KeyStore password to use with SSL
+sslKeyStorePassword: ""
+
+# Controls which particular protocols for secure connection are enabled for handshake
+sslEnabledProtocols: ""
+
+# Controls which particular SSL cipher suites are enabled for secure connection
+sslCipherSuites: ""
+
+# Enables fields ordering in raw message by dictionary
+orderingFields: false
+
+# Check the next expected target SeqNum against the received SeqNum. Default is checked.
+# If a mismatch is detected, apply the following logic:
+# * if lower than expected SeqNum , logout
+# * if higher, send a resend request
+validateSequenceNumbers: true
+
+# It is required param.
+# Socket port for listening to incoming connections from client.
+socketAcceptPort: 1234 
+
+# Session setting that causes the session to reset sequence numbers when initiating a Logon
+resetOnLogon: false
+
+# Tells session whether or not to expect a data dictionary.
+# You should always use a DataDictionary if you are using repeating groups.
+useDataDictionary: true
+
+# Class name of the FIX Server application
+applicationClassName: com.exactpro.sf.services.fix.FIXTESTApplication
+
+# Tells application to keep messages in memory and store them to DB or not
+keepMessagesInMemory: true
+
+# if TRUE then server will not wait logout message from system and will close session immediately after sending its own logout
```

Comment:
if **it is set to** TRUE **,** then **the** server will not wait logout message from **the** system and will close **the** session immediately after sending its own logout